src/Runner.Common/Constants.cs

```diff
@@ -197,6 +197,7 @@ public static class Hooks
             public static readonly string JobStartedStepName = "Set up runner";
             public static readonly string JobCompletedStepName = "Complete runner";
             public static readonly string ContainerHooksPath = "GITHUB_ACTIONS_RUNNER_CONTAINER_HOOKS";
+            public static readonly string CustomHooksPath = "ACTIONS_RUNNER_CONTAINER_HOOK";
```

Comment:
Let's use `ContainerHooksPath` (might not have existed when you forked at first :) )